go/client/cmd_wot_list.go

```diff
@@ -29,28 +29,38 @@ func newCmdWotList(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comma
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(cmd, "list", c)
 		},
-		Flags: []cli.Flag{},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "voucher",
+				Usage: "Only get vouches written by me",
```

Comment:
```suggestion
				Name:  "by-me",
				Usage: "Get vouches written by me",
```

Reply:
i don't care if it's `voucher` or `by-me`. but it looks like you expect `by-me` below.

Reply:
changed to voucher